swim/labels.go

```diff
@@ -145,6 +131,24 @@ func (lo LabelOptions) validateLabels(current map[string]string, additional map[
 	return nil
 }
 
+func countLabels(labels map[string]string, countPrivate bool) int {
```

Comment:
Currently there is no reason to count private labels and one could use `len(labels)` in the code when there is a need, it was more due to naming but I will rename the function to `countNonPrivateLabels`.